src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/BreakpointEvidence.java

```diff
@@ -83,6 +84,23 @@ public String toString() {
         return location.toString() + "^" + weight;
     }
 
+    /**
+     * Returns string representation of BreakpointEvidence in tab-separated form:
+     * Contig[begin:end]    weight  validated   EvidenceType    distal_targets
+     * distal_targets is a (; separated) list of string representations (.toString()) for each distal target
+     *     it is empty if there are no distal targets
+     * Child classes may extend stringRep() by appending class-specific tab-separated info
+     */
+    public String stringRep(final ReadMetadata readMetadata, final int minEvidenceMapq) {
+        final String dtString =  hasDistalTargets(readMetadata, minEvidenceMapq) ?
+                getDistalTargets(readMetadata, minEvidenceMapq).stream()
+                    .map(strandedInterval -> strandedInterval.getInterval().toString() + (strandedInterval.getStrand() ? "1" : "0"))
+                    .collect(Collectors.joining(";"))
+                : "";
+        return location.toString() + "\t" + weight + "\t" + (validated ? "1" : "0") + "\t" + this.getClass().getSimpleName()
+                + "\t" + dtString;
```

Comment:
This will print a trailing tab on evidence that has no distal targets.  Maybe move that tab up to line 96:
"\t" + getDistalTargets...
Yes, I'm being a fussbudget.

Reply:
I actually want the extra tab because it makes parsing easy. I split the string by tabs and then the list of distal targets is the 5th word split by ';'. If that's empty there are no distal targets.

Reply:
Fine with me.